test/hello_world_test.cc

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include "gtest/gtest.h"
+#include "lib/hello_world.h"
+
+/*
+    Tests the HelloWorld class to return the expected string
+*/
+TEST(HelloWorldClass, ShouldReturnHelloWorldString){
```

Comment:
Should we also wrap this test in a namespace? https://google.github.io/styleguide/cppguide.html#Namespaces
The Abseil tutorial also uses namespaces https://github.com/abseil/abseil-hello

Reply:
That's a good idea. Would be really useful once we start adding more code. What do other folks think about it? @w-michal @toadchild @Ralith

Reply:
+1 on using namespaces

Reply:
+1 for namespaces, and AFAIK gtest documentation recommends that test should be in anonymous namespace (but that is not a requirement)

Reply:
Added an anonymous namespace for testing